checker/src/call_visitor.rs

```diff
@@ -973,10 +973,12 @@ impl<'call, 'block, 'analysis, 'compilation, 'tcx, E>
                 trace!("post env {:?}", self.block_visitor.bv.current_environment);
                 return;
             } else {
-                let saved_callee_def_id = self.callee_def_id;
-                self.callee_def_id = def_id;
-                self.deal_with_missing_summary();
-                self.callee_def_id = saved_callee_def_id;
+                if self.block_visitor.bv.check_for_errors {
+                    let saved_callee_def_id = self.callee_def_id;
+                    self.callee_def_id = def_id;
+                    self.deal_with_missing_summary();
```

Comment:
On line 988, there is another `deal_with_missing_summary` call that is not guarded by `check_for_errors`.